en/docs/quick-start-guides/deploy-your-first-integration.md

```diff
@@ -0,0 +1,198 @@
+# Deploy Your First Integration
+
+## Overview
+
+In this guide, you will:
+
+- Create a simple Integration as an API that calls a backend service and returns its response to the client. 
+- The backend service responds with a JSON with a key `message` and the value `Hello World!!!`, and the API deployed in Devant forwards this response to the client.
+- Use Micro Integrator (MI) to develop the API.
+- Push the integration to Devant which automatically builds and deploys the integration to the development environment.
+- Test the integration.
+
+## Prerequisites
+
+1. GitHub account: [Create a GitHub account](https://github.com/signup) if you don't have one already.
+2. If you're signing in to the Devant for the first time, create an organization:
+    1. Go to [https://console.devant.dev/](https://console.devant.dev/) and sign in using your Google, GitHub, or Microsoft account.
+    2. Enter a unique organization name. For example, `Stark Industries`.
+    3. Read and accept the privacy policy and terms of use and Click **Create**.
+3. VSCode: [Install VSCode](https://code.visualstudio.com/download) if you don't have it installed already.
+
+## Step 1: Attach a Git repository
+1. Go to [https://console.devant.dev/](https://console.devant.dev/) and sign in. This opens the organization overview page and lists the projects.
+2. Select the **Default** project.
+3. On the project overview page, click **Attach a Git Repository**.
+4. Click **Authorize with GitHub** to connect Devant to your GitHub account. If you haven't connected your GitHub repository to Devant, authorize the WSO2 cloud app stage with your GitHub account [WSO2 Cloud App](https://github.com/marketplace/choreo-apps).
+5. Under the **Organization**  dropdown click **+ Add**. This redirects you to **Install WSO2 Cloud App Stage** page.
+6. Select your GitHub account and install [WSO2 Cloud App](https://github.com/marketplace/choreo-apps)
+
+    !!! note
+        The **WSO2 Cloud App** requires:
+        - Read and write access to code and pull requests.
+        - Read access to issues and metadata.
+
+         You can [revoke access](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/reviewing-your-authorized-integrations#reviewing-your-authorized-github-apps) at any time. Write access is used to push changes directly to your repository.
+
+7. Select your organization under the **Organization**  dropdown. If your organization is still not listed click the Refetch button.
```

Comment:
```suggestion
7. Select your organization under the **Organization**  dropdown. If your organization is still not listed click the `Refetch` button.
```

Reply:
We only add code for values to the fields or code segments. Buttons we add bold. But in this case, there is no button with Refetch word. It's just an icon. That is why I keep it as plain